packages/rancher-istio-1.8/charts/templates/istio-cni-psp.yaml

```diff
@@ -0,0 +1,51 @@
+{{- if .Values.global.rbac.pspEnabled }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: psp-istio-cni
+  namespace: {{ template "istio.namespace" . }}
+spec:
+  allowPrivilegeEscalation: true
+  fsGroup:
+      rule: RunAsAny
+  hostNetwork: true
+  runAsUser:
+      rule: RunAsAny
+  seLinux:
+      rule: RunAsAny
+  supplementalGroups:
+      rule: RunAsAny
+  volumes:
+  - secret
+  - configMap
+  - emptyDir
+  - hostPath
```

Comment:
Going to trust that this is all minimum privilege requirements for various PSP's. Where did these all come from, made up by hand or does istio publish them ?

Reply:
Istio does not publish this, but https://banzaicloud.com/blog/istio-operator-psp/ this was used as a blueprint to determine which privileges were needed. CNI PSP has the most privileged permissions of the three PSPs being added.

Reply:
Ok, we just need to be careful using Banzai's stuff as theirs is diff from upstream.